### PR TITLE
feat: Update the icons in dashboard/chart context menus to 'Tabler' ones

### PR DESCRIPTION
**Pull request by jayanand05**
Closes: #5159

### Description:
Updated the icons in dashboard/chart context menus to 'Tabler' ones.

Chart context menu
F727BB36-3386-4C5F-848B-EE5CC69D93DC_1_201_a

Dashboard context menu
6FE8F8F4-12D1-4F98-8245-8F7768C9BCE5_1_201_a

### CODE REVIEW

**Comment by jayanand05** [comment]
small update in the icon of Add to dashboard and Move to Space(both in chart and dashboard)
01A2FAFF-0557-45D0-980B-998475A7334B_1_201_a

File: packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx [hunk 1]
```diff
@@ -8,7 +8,17 @@ import {
 } from '@blueprintjs/core';
 import { MenuItem2, Popover2 } from '@blueprintjs/popover2';
 import { subject } from '@casl/ability';
-import { IconDots, IconPencil } from '@tabler/icons-react';
+import {
```
Comment (rephus) [inline packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx:11]: we could also use                                 <MantineIcon
 which is a wrapper with a theme, this is normally used when we need to apply a different color or something. 
 
 But using the icon directly works here too

**Review by rephus — COMMENTED** [review]
Code looks good, if @PriPatel is happy ,we can merge

**Review by PriPatel — APPROVED** [review]
Looks good - thank you so much !

**Pull request merged**